main.rb

```diff
@@ -0,0 +1,50 @@
+require './app'
+
+class Main
+  def menu_options
+    "
+      1 - list all books
+      2 - list all music albums
+      3 - list all games
+      4 - list all genres
+      5 - list all labels
+      6 - list all authors
```

Comment:
- [ ] I tried to list all authors by selecting optional 6, but the app started to request my first/last name, this is a bug that should be fixed. Kindly work on this and test if it works before submitting it for another review.

![list-authors](https://github.com/josuerushanika/ruby_capstone/assets/53124548/7a3e37f7-3dd2-444b-b364-55c02280390c)

Reply:
It's fixed